Contribute/content/provide-feedback.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Provide feedback for Microsoft Learn content
+description: Learn how to provide feedback to Microsoft Learn content teams using the new feedback experience for documentation and training content.
+author: carlyrevier
+ms.author: cahublou
+ms.topic: contributor-guide
+ms.prod: non-product-specific
+ms.custom: external-contributor-guide
+ms.date: 02/22/2024
+---
+
+# Provide feedback for Microsoft Learn content
+
+This article explains how to provide feedback for Microsoft Learn content using the new feedback experience. This unified feedback experience is now standard across all products that publish documentation or training content on Microsoft Learn. The improved system:
+
+- Simplifies our procedures to efficiently incorporate your feedback.
+- Allows for feedback in languages other than English.
+- Enhances the ability of content authors to respond to feedback.
+- Supports global accessibility for non-English customers.
+- Reduces the burden of providing feedback for the non-developer community.
+
+Depending on the repository, you may see a standard feedback experience and/or an open-source experience.
+
+## Prerequisites
+
+- None, if using the [standard experience](#use-the-standard-experience).
+- A GitHub account, if using the [open-source experience](#use-the-open-source-experience).
+
+## Use the standard experience
+
+The majority of content sets use the standard experience, which allows you to provide feedback using a thumbs up/thumbs down control. You don’t need to log in or create an account to use this experience.
+
+While GitHub issues are no longer supported within repositories using the standard experience, you can still contribute to Microsoft Learn documentation by [creating a pull request](how-to-write-quick-edits.md) if you spot an error or wish to add something.
+
+To leave feedback with the standard experience:
+
+1. Navigate to the article you’d like to give feedback on.
+1. Find the **Feedback** link and select it.
+    1. In documentation, the link is at the top of the page and in a larger box at the bottom of the page.
+
+        ![Screenshot of the feedback link at the top of a documentation page.](media/provide-feedback/feedback-link.png)
+
+    1. In training modules, the link is at the top of the module’s landing page and at the bottom of each unit within the module.
+
+        ![Screenshot of the feedback link at the top of a training module.](media/provide-feedback/feedback-link-training.png)
+
+1. Choose the thumbs up option if you found the page helpful; choose the thumbs down option if it wasn’t.
+1. Choose a reason for your feedback. Optionally, use the comment box to provide specific feedback on the content.
```

Comment:
can we add an extra note, encouraging comment text, in particular if they thumbs down.  we could even note the Thumbs controls up to 999 characters of feedback.